src/omotes_simulator_core/entities/assets/esdl_asset_object.py

```diff
@@ -78,8 +79,18 @@ def get_return_temperature(self, port_type: str) -> float:
         raise ValueError(f"No port found with type: {port_type} for asset: {self.esdl_asset.name}")
 
     def get_port_ids(self) -> list[str]:
-        """Get the port ids of the asset."""
-        return [port.id for port in self.esdl_asset.port]
+        """Returns a sorted list of the port ids of the asset."""
+        list_of_ports = sort_ports(
```

Comment:
Why don't you directly put it in the list_of_ports? And shouldn't this object return a list of Ports?

Reply:
Fine, my comment was about the part: 

list_of_ports = ... 

return list_of_ports.

For understanding and readability this is fine.